catalogue/webapp/model/iiif.ts

```diff
@@ -84,6 +84,29 @@ export type IIIFMediaElement = {
   service?: AuthService | AuthService[];
 };
 
+export type IIIFLabelV3 = Record<string, string[]>;
+
+export type IIIFMediaElementV3 = {
```

Comment:
Where did we get to re importing these from the "official" definitions?

Reply:
Just done